### PR TITLE
chore(docs): update graph service

### PR DESCRIPTION
**Pull request by enddynayn**
Update Graph Service documentation so that
it reflects the latest changes.

### CODE REVIEW

**Review by claireolmstead — APPROVED** [review]
Read through it and it looks good to me!!! 🚢

File: developer-docs/graph/README.md [hunk 1]
```diff
@@ -1,285 +1,155 @@
...
+3. Set up environment variables:
+   ```bash
+   cp env-files/graph.template.env .env
+   ```
+   Edit the `.env` file with appropriate values for your environment.
 
-Note: using [docker compose file](docker-compose.yaml) to start the services. This will start the services in development mode.
+4. Start all services using Docker Compose:
+   ```bash
+   docker compose up -d redis frequency graph-service-api graph-service-worker
```
Comment (aramikm) [inline developer-docs/graph/README.md:67]: I don't think works. It looks like we've 2 instances of redis
- redis
- bull

I'm unaware why we've split them in the first place but that might be the issue

Comment (aramikm) [reply developer-docs/graph/README.md:67]: you might also need to run the bull instance on the requested port

Comment (aramikm) [reply developer-docs/graph/README.md:67]: Actually this seem to work the problem was that when you don't have a registered provider the worker just doesn't error out and just hangs there and that's something that I think we have a ticket for

File: developer-docs/graph/README.md [hunk 2]
```diff
@@ -1,285 +1,155 @@
...
+   ```bash
+   docker compose up -d redis frequency
+   ```
 
-```sh
-make setup
-```
+2. Set up the graph with test data:
+   ```bash
+   make setup-account
```
Comment (aramikm) [inline developer-docs/graph/README.md:81]: I think you would need to run this in both cases (locally and docker) to have a registered provider with capacity.

File: developer-docs/graph/README.md [hunk 3]
```diff
@@ -1,285 +1,155 @@
...
+4. Start the Graph Service Worker:
+   ```bash
+   make start-graph-worker
+   ```
 
-#### 4. Check the job in [BullUI](http://0.0.0.0:3000/queues/), to monitor job progress based on defined tests.
+### Verification
 
-## 📋 Testing
+- Access Swagger UI: [http://localhost:3000/docs/swagger](http://localhost:3000/docs/swagger)
```
Comment (aramikm) [inline developer-docs/graph/README.md:96]: the ports would be different when we run it inside docker

Comment (enddynayn) [reply developer-docs/graph/README.md:106]: Good catch. I updated the documentation.

**Pull request merged**